localenv/mock-account-servicing-entity/app/lib/parse_config.server.ts

```diff
@@ -68,5 +69,6 @@ export const CONFIG: Config = {
   key: loadOrGenerateKey(process.env.KEY_FILE),
   publicHost: process.env.PUBLIC_HOST ?? '',
   testnetAutoPeerUrl: process.env.TESTNET_AUTOPEER_URL ?? '',
-  authServerDomain: process.env.AUTH_SERVER_DOMAIN || 'http://localhost:3006'
+  authServerDomain: process.env.AUTH_SERVER_DOMAIN || 'http://localhost:3006',
+  peeringAsset: process.env.PEERING_ASSET || 'USD'
```

Comment:
I was thinking of this going into the respective `seed.yml` files (so you'd be able to set the default peering asset in the same location all of the assets are defined), but this works as well